toolkit/tools/internal/file/file.go

```diff
@@ -86,6 +86,15 @@ func CopyAndChangeMode(src, dst string, dirmode os.FileMode, filemode os.FileMod
 	return copyWithPermissions(src, dst, dirmode, true, filemode)
 }
 
+// Write writes a string to the file dst.
```

Comment:
Forgot to update the comment:
```suggestion
// Read reads a string from the file src.
```

Reply:
fixed